overlord/snapstate/snapstate.go

```diff
@@ -2103,6 +2103,27 @@ func finalizeUpdate(st *state.State, tasksets []*state.TaskSet, hasUpdates bool,
 	return tasksets
 }
 
+func reRefreshSummary(updated []string, flags *Flags) string {
+	var msg string
+	n := len(updated)
+	if n > 1 && !flags.IsAutoRefresh {
+		n = 2
+	}
+	switch n {
+	case 0:
+		return ""
+	case 1:
+		msg = fmt.Sprintf(i18n.G("Monitoring snap %q for need of extra refresh steps"), updated[0])
+	case 2, 3:
+		quoted := strutil.Quoted(updated)
+		// TRANSLATORS: the %s is a comma-separated list of quoted snap names
+		msg = fmt.Sprintf(i18n.G("Monitoring snaps %s for need of extra refresh steps"), quoted)
+	default:
+		msg = fmt.Sprintf(i18n.G("Monitoring the %d snaps for need of extra refresh steps"), len(updated))
```

Comment:
It reads a bit nicer without `the`.
```suggestion
		msg = fmt.Sprintf(i18n.G("Monitoring %d snaps for need of extra refresh steps"), len(updated))
```

Reply:
Also is this message hit by a unit test?

Reply:
While I think the `for need of extra refresh steps` does makes sense, I don't think it's easy to understand. And do we need `snaps`? What happens if its a single snap? `snap(s)`?

How about `Monitoring %d to determine whether extra refresh steps are required.`

Reply:
If it was a single snap, it would hit the case above in line 2116. I like the suggestion, with a small preference for keeping `snaps` (e.g., `monitoring 4 snaps to determine ...`). @pedronis if you're ok with that, I can push this PR along

Reply:
yes, thx, please go ahead